openghg/util/_util.py

```diff
@@ -203,7 +203,7 @@ def verify_site(site: str) -> Optional[str]:
     else:
         site_code = site_code_finder(site_name=site)
         if site_code is None:
-            logger.warning(f"Unable to find site code for {site}, please provide additional metadata.")
+            logger.info(f"Unable to find site code for {site}, please provide additional metadata.")
```

Comment:
Why has this changed some warning to info?

Reply:
Its picked from the changes suggested by Rachel

Reply:
For reference, in my solution I had to change some of the logger.warning to logger.info to make sure this wasn't sending to the StreamHandler. Fine to keep this as logger.warning in the new solution.

Reply:
Updated back to warning